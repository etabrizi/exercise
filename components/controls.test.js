import * as React from 'react';
import Controls from './Controls';
import renderer from 'react-test-renderer';

const Props = {
    counterDecrease: () => { },
    counterIncrease: () => { },
    toggleInfo: true,
    gender: 'male',
    toggleGender: () => { },
    infoToggle: () => { },
    searchToggle: true,
    toggleSearch: () => { },
    closeAll: () => { }
}

describe('Controls component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Controls {...Props} />)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});







