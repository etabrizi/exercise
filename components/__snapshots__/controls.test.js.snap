// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls component renders correctly 1`] = `
<nav
  className="jsx-782766476"
>
  <ul
    className="jsx-782766476 "
  >
    <li
      className="jsx-782766476"
    >
      <a
        className="jsx-782766476"
        href="#"
        onClick={[Function]}
      >
        <span
          className="jsx-782766476 icon-circle-left"
        />
      </a>
    </li>
    <li
      className="jsx-782766476"
    >
      <a
        className="jsx-782766476"
        href="#"
        onClick={[Function]}
      >
        <span
          className="jsx-782766476 icon-circle-right"
        />
      </a>
    </li>
    <li
      className="jsx-782766476"
    >
      <a
        className="jsx-782766476"
        href="#"
        onClick={[Function]}
      >
        <span
          className="jsx-782766476 icon-male1"
        />
      </a>
    </li>
    <li
      className="jsx-782766476 "
    >
      <a
        className="jsx-782766476"
        href="#"
        onClick={true}
      >
        <span
          className="jsx-782766476 icon-info"
        />
      </a>
    </li>
    <li
      className="jsx-782766476 "
    >
      <a
        className="jsx-782766476"
        href="#"
        onClick={[Function]}
      >
        <span
          className="jsx-782766476 icon-search"
        />
      </a>
    </li>
    <li
      className="jsx-782766476 hide"
      onClick={[Function]}
    >
      <a
        className="jsx-782766476"
        href="#"
      >
        <span
          className="jsx-782766476 icon-cancel-circle"
        />
      </a>
    </li>
  </ul>
</nav>
`;
